x/project/internal/types/msgs.go

```diff
@@ -160,9 +161,9 @@ func (msg MsgUpdateProjectStatus) ValidateBasic() sdk.Error {
 
 	// Check that DIDs valid
 	if !did.IsValidDid(msg.ProjectDid) {
-		return did.ErrorInvalidDid(DefaultCodespace, "project did is invalid")
+		return sdkerrors.Wrap(did.ErrorDidPubKeyMismatch, "project did is invalid")
```

Comment:
Incorrect error type, please use the original one (`ErrorInvalidDid`)

Reply:
removed